README.md

```diff
@@ -85,6 +85,28 @@ Please run the following command to download demos required models if you run th
 npm run fetch-models
 ```
 
+### Breaking Changes
+
+WebNN is a living specification and still subject to breaking changes, which may impact the samples depending on your browser version. The following are recent:
+
+- 2024-08-?? ⏳ Add `MLNumber` to `constant`, `clamp`, `pad` - [spec change](https://github.com/webmachinelearning/webnn/pull/647), [Chromium change](https://chromium-review.googlesource.com/c/chromium/src/+/5466806)
```

Comment:
Would this be a API break change? I mean whether it needs user code change once it is implemented?

Reply:
Ooh, good realization. *Technically* it's an API change, but if BigInt and JS Number implicitly both convert to MLNumber transparently, then the change would be silent for the caller.